dataplane/kernel_interface_handler.cpp

```diff
@@ -96,6 +101,27 @@ KernelInterfaceWorker::KernelInterfaceWorker(const KernelInterfaceWorkerConfig&
 	}
 }
 
+KernelInterfaceWorker::KernelInterfaceWorker(KernelInterfaceWorker&& other)
+{
+	*this = std::move(other);
+}
+
+KernelInterfaceWorker& KernelInterfaceWorker::operator=(KernelInterfaceWorker&& other)
+{
+	size_ = std::exchange(other.size_, 0);
+	for (std::size_t i = 0; i < size_; ++i)
+	{
+		std::swap(phy_ports_[i], other.phy_ports_[i]);
+		std::swap(phy_queues_[i], other.phy_queues_[i]);
+		std::swap(forward_[i], other.forward_[i]);
+		std::swap(in_dump_[i], other.in_dump_[i]);
+		std::swap(out_dump_[i], other.out_dump_[i]);
+		std::swap(drop_dump_[i], other.drop_dump_[i]);
+	}
+	port_mapper_ = std::move(other.port_mapper_);
+	return *this;
+}
+
```

Comment:
No point in defining any of this, those are `std::array`'s, they are not benefiting from moves, they will just be copied.
So delete this